readthedocs_ext/readthedocs.py

```diff
@@ -22,7 +21,62 @@
 ]
 
 
+def finalize_media(app):
+    """
+    Point media files at our media server.
+    """
```

Comment:
Not a huge priority, I opened #14 to deal with it later. I'm sure there will be some cleanup required as well